react/features/subtitles/components/ClosedCaptionButton.native.js

```diff
@@ -31,9 +31,7 @@ class ClosedCaptionButton
  * @returns {Props}
  */
 export function mapStateToProps(state: Object, ownProps: Object) {
-    const { transcribingEnabled } = state['features/base/config'];
-    const { isGuest = true } = state['features/base/jwt'];
-    const enabled = getFeatureFlag(state, CLOSE_CAPTIONS_ENABLED, true) && transcribingEnabled && !isGuest;
+    const enabled = getFeatureFlag(state, CLOSE_CAPTIONS_ENABLED);
```

Comment:
We need some extra changes here so `visible` also follows the logic in abstractMapStateToProps, except when the button is not enabled.